src/Core/MySQL/MySQLReplication.h

```diff
@@ -368,7 +368,8 @@ namespace MySQLReplication
     {
         QUERY_EVENT_DDL = 0,
         QUERY_EVENT_MULTI_TXN_FLAG = 1,
-        QUERY_EVENT_XA = 2
+        QUERY_EVENT_XA = 2,
+        QUERY_EVENT_OTHER = 3
```

Comment:
Could you call this `QUERY_SAVEPOINT` please? We are using that name in our fork (to be merged upstream soon), good to avoid unnecessary naming conflicts :-)

Reply:
ok